### ISSUE TITLE
New Ruby Error in Container as of 15-Aug-2022 Update

### ISSUE THREAD

**Issue by anthturner** [crazy-max/docker-linguist#29]
### Behaviour
Running the following inside a GitHub Action (ubuntu runner) causes a Ruby error. **This began occurring as of yesterday's update to the container.**

Command:
```
LANGS=$(docker run -t --rm -v "$GITHUB_WORKSPACE:/repo" crazymax/linguist:latest --json)
```

Output:
```
+ LANGS='/usr/lib/ruby/gems/3.1.0/gems/github-linguist-7.22.1/bin/github-linguist:48:in `new'\'': repository path '\''/repo/'\'' is not owned by current user (Rugged::ConfigError)
	from /usr/lib/ruby/gems/3.1.0/gems/github-linguist-7.22.1/bin/github-linguist:48:in `github_linguist'\''
	from /usr/lib/ruby/gems/3.1.0/gems/github-linguist-7.22.1/bin/github-linguist:142:in `<top (required)>'\''
	from /usr/bin/github-linguist:25:in `load'\''
	from /usr/bin/github-linguist:25:in `<main>'\''
```

#### Steps to reproduce this issue
Run command above from within a GitHub Action.

#### Expected behaviour

> Tell me what should happen
$LANGS is populated with the JSON output for `linguist`.

#### Actual behaviour

> Tell me what happens instead
See error code above. Causes the step to fail with exit code 1.

**Comment by crazy-max** [comment]
Oh right good catch! In the meantime you can switch back to `crazymax/linguist:4.20.0`.

**Comment by crazy-max** [comment]
I will add some e2e tests to avoid this issue in the future.

**Comment by anthturner** [comment]
@crazy-max Did you mean `7.20.0`? `4.20.0` isn't an existing tag. :)

**Comment by crazy-max** [comment]
My bad yes `razymax/linguist:7.20.0`

**Comment by crazy-max** [comment]
@anthturner Fixed and test added in #31, see https://github.com/crazy-max/docker-linguist/runs/8293286551?check_suite_focus=true#step:7:1
